Makefile

```diff
@@ -1,9 +1,11 @@
-deps: composer.json
+.PHONY: deps
+deps: composer.json composer.lock
```

Comment:
This way it will run `composer install` on every build.

If it'd be like this, it would only run `composer update` if `composer.json` is newer than `composer.lock`:
```
deps: composer.lock
composer.lock: composer.json
    composer update
```

Reply:
Yes, so my question was about whether this was intended as from what I see only 1 `composer install` call is enough. Sure the overhead is small but it's not like removing that overhead is costing a lot of efforts here either